### PR TITLE
added self & jupyterlab documentation to help package

### LINKED ISSUE #3036: Jupyterlab Help
Please add a JupyterLab Reference to the Help menu. All I see is iPython and Notebook.

### PR DESCRIPTION
**Pull request by jzf2101**
closes #3036

### CODE REVIEW

**Comment by blink1073** [comment]
Ha, I didn't realize "self" meant you until I read the diff.

File: packages/help-extension/src/index.ts [hunk 1]
```diff
@@ -73,6 +73,10 @@ const HELP_CLASS = 'jp-Help';
 
 const RESOURCES = [
   {
+    text: 'Jupyterlab Reference',
+    url: 'http://jupyterlab.readthedocs.io/en/stable/'
```
Comment (blink1073) [inline packages/help-extension/src/index.ts]: Mind changing this to `https`?  We can't embed `http` if jlab itself is served via `https`.

**Comment by blink1073** [comment]
Thanks!

File: packages/help-extension/src/index.ts [hunk 2]
```diff
@@ -73,6 +73,10 @@ const HELP_CLASS = 'jp-Help';
 
 const RESOURCES = [
   {
+    text: 'Jupyterlab Reference',
```
Comment (ellisonbg) [inline packages/help-extension/src/index.ts]: caps - `JupyterLab`

**Pull request merged**